src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/DefaultCaseDataAccessControl.java

```diff
@@ -233,4 +251,22 @@ private static boolean isUserAllowedAccessToCase(GrantType grantType) {
             .map(roleAssignment -> GrantType.valueOf(roleAssignment.getGrantType()))
             .collect(Collectors.toList());
     }
+
+    @Override
+    public boolean shouldItRemoveCaseDefinitionBaseOnRoleType(Set<AccessProfile> accessProfiles,
+                                                              Predicate<AccessControlList> access,
+                                                              String caseTypeId) {
+        // In R.A if the access is create the RoleType has to be organisation.
+        final var accessProfile = generateCreationAccessProfilesByCaseTypeId(caseTypeId);
```

Comment:
why var is used here?

Reply:
Because it helps to reduce code line length and avoid checkstyle indentation.